src/util/ChartUtils.js

```diff
@@ -873,6 +873,8 @@ export const parseSpecifiedDomain = (specifiedDomain, dataDomain, allowDataOverf
     const value = +MIN_VALUE_REG.exec(specifiedDomain[0])[1];
 
     domain[0] = dataDomain[0] - value;
+  } else if (toString.call(specifiedDomain[0]) === '[object Function]') {
```

Comment:
How about use _.isFunction to detect whether `specifiedDomain[0]` is a function?